src/common/src/config.rs

```diff
@@ -1166,6 +1168,10 @@ pub mod default {
             true
         }
 
+        pub fn check_fast_compaction_result() -> bool {
```

Comment:
Prefer to set the default value as false If it will cause obvious performance regression